robot/board.py

```diff
@@ -81,8 +81,8 @@ def _recv(self, should_retry=True):
                 return b''
 
             self.data += message
-        line = self.data.split(b'\n', 1)
-        self.data = self.data[len(line):]
+        line = self.data.split(b'\n', 1)[0]
```

Comment:
Maybe add a comment here about why we're not using unpacking for this (which feels like the more natural solution)?

(I also wondered about using `*args`, but I'm not sure doing so ends up any clearer than what's already here)